### DISCUSSION TITLE
Presentation for 11/19/24

### DISCUSSION THREAD

**Discussion by DylanTh20** [ecn310/course-project-taxes-tariffs#21] [General]
### **_Information about our data:_**
- 1988 to 2022
- All three raw datasets provided by the World Integrated Trade Solutions and stored in excel 
- The data specifically covers International trade as a percent of revenue, Value Added Tax as a percent of revenue, and GDP per capita in current USD. The data has been transformed from wide to long, and all three of these datasets have been merged together. Each observation is given an ID of 1 or 2, 1 is given to developed countries that have a GDP per capita of 12000 USD or higher and 2 is given to developing countries that have a GDP per capita of 12000 USD or less. 
- Our focus is to compare how things differ between develop and developing countries in terms of the relationship each country classification has between taxes and tariffs. 
- So far we have found that developing countries have a greater negative correlation between taxes and tariffs, but more tests need to be conducted in order to verify that this correlation is correct.

**Comment by umarbilgrammi** [comment]
### **Updates** 
- outliers seychelles and Luxembourg b/c they have a small population making gdp per capita higher causing them to be included in the developed countries tab....
- within our current system of classifying countries Spain is a developing country
- we are using current usd as of 2022 and cutting gdp per capita off at 12,000 
- we found the outlier on the 2 way histogram is china at -15.8 while the second lowest value is around -0.8. (international tax as % of revenue )

**Comment by DylanTh20** [comment]
### **China's Outlier Point in Greater Detail**

![Taxes on international trade (% of revenue)](https://github.com/user-attachments/assets/447b4740-0c0c-48c3-a2bc-ecc927d0af79)

To find this image, I navigated to the indicator Taxes on international trade (% of revenue), then limited the years from 2000 to 2010. That produced the graph above, clearly displaying that this must be bad data entry. China experienced one of it's best years of economic growth, with real GDP increasing by 9.3% in 2005, (the year this bad data entry was made) meaning this data entry must be a mistake.

**Comment by umarbilgrammi** [comment]
### **Summary Statistics and Data Overview**

### **Total Dataset Summary Statistics**

_Variables:_
- year --> 6755 observations min and max 1988 - 2022
- gdp --> 6500 observations 
- vat --> 3212 observations 
- international --> 2970 observations 

### **Means:** 

- gdp: 11318.85 dollars 
- vat: 10.26734 (% of revenue) 
- international: 10.23171 (% of revenue) 


### **Developed Countries Dataset** 

- Minimum and Maximum for year - 1988 - 2022
2041 observations for year --> 1786 observations for gdp --> 1,100 for vat --> 741 international 

- mean gdp: 33928.8 (do we need standard deviation for final report?)
- mean vat: 11.61037
- mean international: 3.835927

### **Developing Countries Data Set:**

- min and max for years are 1988 and 2022 
4714 observations for year, 4714 for gdp, 2,112 for vat, 2,229 for international 
- essentially our data analysis for developing countries has a bigger data set and thus would be more accurate 
- mean gdp: 2991.279
- mean vat: 9.552835
- mean international: 12.35789


### **Correlation Tests** 
- _Developed countries_: pwcorr international vat signficance test gives -0.2509 with a p-value of 0.000 --> reject null for developed countries 

- _Developing countries_: pwcorr international vat significance test gives -0.3421 with a p-value of 0.000 --> reject null hypothesis 
- there is a stronger negative correlation between value added tax and tarriffs in developing countries in comparison to developed countries 

- _total data set_: pwcorr test for p-value gives -0.3250 with a p-value of 0.000 --> reject null hypothesis so far 

Note: 3 p-values of 0.000 seems very suspicious, are we doing something wrong?

**Comment by umarbilgrammi** [comment]
### **Graphs** 

![developedhistogram](https://github.com/user-attachments/assets/bfa9182c-d59e-4589-843a-6cfb00eff3b5)

^developed countries scatter plot


![twowaydeveloped](https://github.com/user-attachments/assets/76a41c0b-3d93-4945-992a-9f58e6a6ed2f)

^two way histogram for developed countries red = international blue = vat



![scatterdeveloping](https://github.com/user-attachments/assets/a0a18b34-4446-4c54-9fe7-35e57a2bdb71)

^scatter plot for developing countries 

![histogramdeveloping](https://github.com/user-attachments/assets/ab1978a2-b7ee-4285-a447-44926487dba4)

histogram for developing countries 

- ran into glitch where the scatter and 2 way histogram for developing countries is the same we are not sure what went wrong and spent around an hour troubleshooting. 

- We want to create histograms and time series analysis but Stata is preventing us since there are multiple values for each year 
- Furthermore, we cannot make histograms or time series for certain countries since they are just listed as observations under the variable CountryName
- Any advice?

**Comment by umarbilgrammi** [comment]
### **Interpretation of Results** 
- The negative correlations and statistically significant P-values mean we reject our null hypothesis: In developing countries, there is no difference in revenue when value added tax or tariffs change.
- This means that our working hypothesis: : If Value added tax or sales tax revenue increases in a developing country, then tariff revenue decreases appears to be true due to the negative correlation seen in the scatterplots
- We have done the bulk of our data analysis we just want to add some time series and specific histograms for certain countries to bolster our arguments 
- This sentiment is also supported by some of the papers we have done research on, such as 
  - Michael, Michael S., Panos Hatzipanayotou, and Stephen M. Miller. "Integrated reforms of tariffs and consumption taxes." Journal of Public Economics 52.3 (1993): 417-428
  - M. Shahe Emran, Joseph E. Stiglitz, "On selective indirect tax reform in developing countries" Journal of Public Economics 599-623 (2005)

**Comment by umarbilgrammi** [comment]
### **What are the next steps** 
- Troubleshoot the histogram and scatterplot issue
- Find out how to make a time series analysis
- Reshape the data to make countries the variable so we can create histograms and scatterplots about over the next few years
- Figure out the years issue when it comes to making histograms

**Comment by umarbilgrammi** [comment]
### **Links:** 
- 424340050b46da2ff8f2989f7213d453abacf2a4
- our new main data set is called: sortedultimatemerge.dta
- do file on how we got to this did not save --> we will recreate it during the reproducibility package
